scripts.py

```diff
@@ -285,6 +290,51 @@ def category_paths(cls, csv_reader):
             csv_reader.fieldnames
         )
 
+    def log_missing_identifiers(self, requested_identifiers, works_qu):
```

Comment:
I added some more detailed logging about the status of the `Work` here, but in general this method was moved into a more general class so it could be used in the `CustomListUploadScript`.